content/content/Usage/apply.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Apply"
+date: 2021-11-16T12:38:01-05:00
+draft: false
+---
+
+The final step of the process is `apply`, all the transformed resources are ready to be applied to destination cluster. This command consumes the output of `transform` command.
+
+```
+crane apply -h
+Apply the transformations to the exported resources and save results in an output directory
+
+Usage:
+  crane apply [flags]
+
+Flags:
+  -e, --export-dir string      The path where the kubernetes resources are saved (default "export")
+  -h, --help                   help for apply
+  -o, --output-dir string      The path where files are to be saved after transformation are applied (default "output")
+  -t, --transform-dir string   The path where files that contain the transformations are saved (default "transform")
+
+Global Flags:
+  -c, --config-file string   Path to config file which contains a yaml representation of cli flags. Explicit flags take precedence over config file values.
+      --debug                Debug the command by printing more information
+
+```
+
+#### Example
+
+```
+crane apply -e export -t transform -o output
+```
+
+Apply the resources from `transform` directory to current context
```

Comment:
Apply the patches in the `transform` directory to the resources in the `export` directory and save the modified resource files in the `output` directory